server/CGameHandler.h

```diff
@@ -310,4 +310,9 @@ class clientDisconnectedException : public std::exception
 
 };
 
+class notAllowedActionException : public std::exception

```

Comment:
Someone also coded such thing a bit higher:
```
class clientDisconnectedException : public std::exception
{

};
```
Anyway I think that it's don't have too much sense create such future-oriented things if they are empty.

Reply:
It's my code as well. I choose to throw another object just for the sake of readability and might be to put logging in there as well instead of spreading it around the code.

@FeniksFire any suggestion on whatever I should throw for just temporary things?

Reply:
@FeniksFire what is in general wrong with declaring new exception class just to filter it on catch?

Reply:
@ArseniyShestakov think that it's mostly depends on situation, so there is no one rule.
@alexvins ehh, such empty things for me makes code harder to read, and maintain the proper quality of the code. Of course there is nothing wrong with that code, just there is no need to code in this way and want point that such things can make trash in future.